core/cloudflow-akka-testkit/src/main/scala/cloudflow/akkastream/testkit/javadsl/AkkaStreamletTestKit.scala

```diff
@@ -79,11 +80,19 @@ object AkkaStreamletTestKit {
  */
 final case class AkkaStreamletTestKit private[testkit] (system: ActorSystem,
                                                         mat: Option[ActorMaterializer] = None,
+                                                        volumeMounts: List[VolumeMount] = List.empty,
                                                         config: Config = ConfigFactory.empty())
     extends BaseAkkaStreamletTestKit[AkkaStreamletTestKit] {
 
   def withConfig(c: Config): AkkaStreamletTestKit = this.copy(config = c)
 
+  def withVolumeMounts(volumeMounts: VolumeMount*): AkkaStreamletTestKit = this.copy(volumeMounts = volumeMounts.toList)
+
+  /**
+   * Java API
+   */
+  def withVolumeMounts(volumeMounts: Array[VolumeMount]): AkkaStreamletTestKit = withVolumeMounts(volumeMounts: _*)
```

Comment:
Java also supports varargs, add   @varargs annotation. (scala.annotation.varargs)
You can see an example of how we use this in StreamletShape.scala. We also chose there to have at least one inlet/ outlet in the withOutlets and withInlets calls, which are conceptually very similar to withVolumeMounts. 
Please see https://github.com/lightbend/cloudflow/blob/27ac2b57888a74f2353bc198ec5c627c68f6123b/core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletShape.scala#L22-L70

Reply:
Done.